From 3ba95d65cc3d6d154241794b537e89b6b25a39e9 Mon Sep 17 00:00:00 2001
From: Alex Wilson <alex.wilson@joyent.com>
Date: Wed, 22 Mar 2017 14:28:13 -0700
Subject: [PATCH] joyent/node-cueball#105 Agent pinger logs at INFO level on
 success Reviewed by: Cody Peter Mello <cody.mello@joyent.com>

---
 .gitattributes | 1 +
 CHANGES.adoc   | 8 ++++++++
 lib/agent.js   | 2 +-
 package.json   | 2 +-
 4 files changed, 11 insertions(+), 2 deletions(-)
 create mode 100644 .gitattributes

diff --git a/.gitattributes b/.gitattributes
new file mode 100644
index 0000000..b061fcd
--- /dev/null
+++ b/.gitattributes
@@ -0,0 +1 @@
+/CHANGES.adoc merge=union
diff --git a/CHANGES.adoc b/CHANGES.adoc
index 0b490d0..1aa5dd5 100644
--- a/CHANGES.adoc
+++ b/CHANGES.adoc
@@ -6,6 +6,14 @@ toc::[]
 
 ## v2.x
 
+### v2.2.3
+
+Maintenance release.
+
+Bugs fixed:
+
+ - #105 Agent pinger logs at INFO level on success. Much logs. Very noise.
+
 ### v2.2.2
 
  - Fix-up for 2.2.1 being broken (lib/agent.js missing)
diff --git a/lib/agent.js b/lib/agent.js
index fbdd00f..74e1d07 100644
--- a/lib/agent.js
+++ b/lib/agent.js
@@ -361,7 +361,7 @@ CueBallAgent.prototype.checkSocket = function (host, handle, socket) {
 				self.log.warn('got a 500, closing');
 				handle.close();
 			} else {
-				self.log.info('health check ok, releasing');
+				self.log.debug('health check ok, releasing');
 				handle.release();
 			}
 		});
diff --git a/package.json b/package.json
index 87cde34..905af66 100644
--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "cueball",
-  "version": "2.2.2",
+  "version": "2.2.3",
   "description": "manage a pool of connections to a multi-node service where nodes are listed in DNS",
   "main": "lib/index.js",
   "dependencies": {
-- 
2.21.0

